socorro/processor/processor_app.py

```diff
@@ -151,7 +160,10 @@ def _transform(self, crash_id):
                         exceptions = exception
                     else:
                         exceptions = [exception]
-                    client = raven.Client(dsn=self.config.sentry.dsn)
+                    client = raven.Client(
+                        dsn=self.config.sentry.dsn,
+                        release=SOCORRO_REVISION,
+                    )
```

Comment:
The code changes in this file are identical to the ones in `socorro.lib.transform_rules`. I think we want to keep them identical. I also think there's some non-trivial work being done. Seems prudent to make this a function in `socorro.lib` somewhere and then use that function in both places to generate a `raven.Client`.